packages/react-percy-webpack/src/resolve/index.js

```diff
@@ -1,9 +0,0 @@
-import detectWebpackConfigPath from './detectWebpackConfigPath';
-import getWebpackConfigExports from './getWebpackConfigExports';
-import requireWebpackConfig from './requireWebpackConfig';
-
-export default function resolve(packageRoot, webpackConfigPath) {
```

Comment:
No longer needing to load users' webpack configs meant I could kill quite a bit of code here